app/views/admin/loans/timeline/table/_group.html.slim

```diff
@@ -0,0 +1,13 @@
+- wrapper_tag = {tag: is_first ? 'div' : 'tr'}
+- column_span = entry.descendants_only_steps? ? max_depth - entry.depth : 1
+
+*wrapper_tag
+  / The root group is not actually displayed
+  - unless entry.root?
+    td rowspan="#{entry.descendant_step_count}" colspan="#{column_span}" data-item="group"
```

Comment:
These data-items have been added to prepare for Redmine issue 5023 and remainder of 5024 (styling of table).